lib/devise_token_auth/controllers/helpers.rb

```diff
@@ -66,14 +66,14 @@ def current_#{group_name.to_s.pluralize}
               end.compact
             end
 
-            def render_authenticate_#{group_name}_error
+            def render_authenticate_error
```

Comment:
I have changed the method name to `render_authenticate_error` .

Exists error-render methods are using simple method name like `render_create_error`.
They don't include #{group_name} and #{mapping} like `render_create_error`.
So new method name also don't include these.

https://github.com/lynndylanhurley/devise_token_auth#registrations-controller